packages/odyssey-react/src/components/Status/Status.module.scss

```diff
@@ -10,17 +10,17 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-$status-indicator-size: 1em * 0.5;
+$status-indicator-size: 1rem * 0.5;
```

Comment:
I'm using this PR to test the sustainability of removing `em` in favor of `rem` across our components. This will likely be reverted before merging.

Since we'll be moving to `calc()` in place of Sass, we can now have mixed values as well.